app/src/main/java/com/example/immadisairaj/quiz/api/ApiCount.java

```diff
@@ -0,0 +1,36 @@
+package com.example.immadisairaj.quiz.api;
+import com.google.gson.annotations.SerializedName;
+
+public class ApiCount {
+
+	@SerializedName("category_id")
+	private int categoryId;
+
+	@SerializedName("category_question_count")
+	private CategoryQuestionCount categoryQuestionCount;
+
+	public void setCategoryId(int categoryId){
+		this.categoryId = categoryId;
+	}
+
+	public int getCategoryId(){
+		return categoryId;
+	}
+
+	public void setCategoryQuestionCount(CategoryQuestionCount categoryQuestionCount){
+		this.categoryQuestionCount = categoryQuestionCount;
+	}
+
+	public CategoryQuestionCount getCategoryQuestionCount(){
+		return categoryQuestionCount;
+	}
+
+	@Override
+ 	public String toString(){
```

Comment:
can you remove this method?
or add this method in the other two classes.
Because we have to follow the same format everywhere.

Reply:
Thank you i already updated it in my local remote. But it seems this pull request not updated

Reply:
I already updated it